hyprbars/README.md

```diff
@@ -24,4 +24,8 @@ plugin {
 
 `bar_text_size` -> (int) bar's title text font size (default 10)
 
-`bar_text_font` -> (str) bar's title text font (default "Sans")
\ No newline at end of file
+`bar_text_font` -> (str) bar's title text font (default "Sans")
+
+`button_max_color` -> (col) maximize button color
```

Comment:
One, "max" is very misleading. I'd just go with `maximize`. Two, go with hyprland style of `col.name`. Furthermore, maybe move button configs to `buttons:` for future-proofing?

Reply:
sounds good, done.